src/lru.js

```diff
@@ -130,6 +126,9 @@ class LRU {
 					next.prev = prev;
 				}
 			}
+			if (resetTtl) {
```

Comment:
@avoidwork This was previously skipped for the last entry, so I moved it out of the `if not last entry` block, as it made no sense to keep it there.

Reply:
That's where it was with previous iteration 👍🏻